src/Message.hpp

```diff
@@ -24,23 +24,51 @@ namespace canbus
         NET_GATEWAY,
         EASY_SYNC
     };
-    
+
+    /** Values used to encode specific flags in the can_id field of Message
+     */
+    enum MessageFlags {
+        FLAG_ERROR = (1 << 30),
+        FLAG_REMOTE_TRANSMISSION_REQUEST = (1 << 30),
```

Comment:
Are FLAG_ERROR and FLAG_REMOTE_TRANSMISSION_REQUEST really equal?

Reply:
Oh man ... Yet another example of the number of errors per line going to infinity when the number of lines goes to zero...

Reply:
Well, if every code needs a bug then the math sounds about right :P